### DISCUSSION TITLE
Open Finance APIs

### DISCUSSION THREAD

**Discussion by puneetlakhina** [bacen/pix-api#564] [Q&A] [answered]
Olá,

Por favor, desculpe meu português ruim, estou usando o Google Tradutor.

Trabalho em um aplicativo onde as pessoas podem conversar e os lojistas podem enviar códigos PIX para os compradores interessados em pagá-los. Estou tentando determinar qual é a melhor maneira de integrar com o modelo de financiamento aberto de fazer PIX. Fundamentalmente, parece que precisamos de 3 peças de funcionalidade:

1. Obtenha a lista de aplicativos do banco
2. Redirecione para o aplicativo do banco com os parâmetros apropriados
3. Obtenha um redirecionamento do aplicativo do banco de volta para o nosso aplicativo se o pagamento for bem-sucedido/falhado/etc.

Existem APIs/especificações abertas para essas três coisas?

O tipo de experiência que estamos tentando construir é semelhante ao destacado por Belvo neste vídeo: https://www.youtube.com/watch?v=98hgLy_nfcQ&list=PLBp3o9hAmq8tl0q5CTfvPviw6o_0xDO2H ( Veja mais detalhes aqui: https://belvo. com/blog/belvo-launches-open-finance-payment-initiation-leverage-pix-brazil/ )

Agradeço antecipadamente por sua ajuda.

---
English 

Hello,

I work on an app where people can chat and merchants can send PIX codes to buyers interested in paying them.  I am trying to determine what is the best way to integrate with the open finance model of doing PIX. Fundamentally it seems like we need 3 pieces of functionality:

1. Obtain List of Bank Apps
2. Redirect to bank app with the appropriate parameters
3. Get a redirect from the bank app back to our app if the payment is successful/failed/etc.

Are there open APIs/specs for these 3 things?

The kind of experience we are trying to build is similar to the one highlighted by Belvo in this video: https://www.youtube.com/watch?v=98hgLy_nfcQ&list=PLBp3o9hAmq8tl0q5CTfvPviw6o_0xDO2H ( See more details here: https://belvo.com/blog/belvo-launches-open-finance-payment-initiation-leverage-pix-brazil/ )

Thanks in advance for your help.
---

**Comment by rubenskuhl** [comment, marked as answer]
API reference for Open Finance can be found at https://openfinancebrasil.atlassian.net/wiki/spaces/OF/overview , which is linked from https://openfinancebrasil.org.br/ .
Note that Open Finance, like Pix, is a regulated environment; one needs to be a Central Bank authorized operator to access such APIs.

**Reply by puneetlakhina** [reply]
Thank you very much for the pointer. Assuming one becomes a central bank authorized operator, what entity (or entities) would provide an implementation for these APIs?

**Reply by rubenskuhl** [reply]
Matera, JD, Celcoin, Rendimento, BTG, Itaú and Iniciador are some of the enablers in the market.
